examples/slices/slices.go

```diff
@@ -5,15 +5,32 @@ package main
 
 import "fmt"
 
+func handleIndexOutOfBoundsPanic() {
```

Comment:
I prefer to leave this function out of the example. The example doesn't have any other functions, and this is not necessary